### ISSUE TITLE
Add Prantik’s lecture to useful resources

### ISSUE THREAD

**Issue by KirstieJane** [ME-ICA/tedana#81]
I just saw this tweet and thought it would be a good idea to link to this talk by Prantik as a useful introduction to multi-echo demonising :smiley:

https://twitter.com/mvlombardo/status/1013681900052008960?s=21

https://www.pathlms.com/ohbm/courses/5158/sections/7788/video_presentations/75977

**Comment by emdupre** [comment]
Yes, it would be great to have this (and other relevant lectures / materials) integrated into the documentation ! Where do you think this would go best ?
